Tasks/Lesson_5/Methodichka_10.java

```diff
@@ -0,0 +1,17 @@
+package Tasks.Lesson_5;
+//Посчитать факториал числа в границах от 10 до 15 (не используя рекурсию).
+
+public class Methodichka_10 {
+    public static void main(String[] args) {
+        int number = 10;
+
+        while (number < 16) {
```

Comment:
I thought about one random number in range from 10 to 15, but all numbers in this range is fine too